scipy/stats/_binned_statistic.py

```diff
@@ -664,3 +632,70 @@ def binned_statistic_dd(sample, values, statistic='mean',
     result = result.reshape(input_shape[:-1] + list(nbin-2))
 
     return BinnedStatisticddResult(result, edges, binnumbers)
+
+
+def _bin_edges(sample, bins=None, range=None):
+
+    Dlen, Ndim = sample.shape
+
+    nbin = np.empty(Ndim, int)    # Number of bins in each dimension
+    edges = Ndim * [None]         # Bin edges for each dim (will be 2D array)
+    dedges = Ndim * [None]        # Spacing between edges (will be 2D array)
+
+    # Select range for each dimension
+    # Used only if number of bins is given.
```

Comment:
this commented line is an atavism from before `_bin_edges` was factored out. The comment doesn't seem to hold anymore, is that correct behavior?

Reply:
I do not understand.

Reply:
I'm asking in particular about the comment:
`# Used only if number of bins is given.`
What if number of bins is not given.

Reply:
The comment is still relevant.
This part is actually not used if the bins are given explicitly.

Reply:
>The comment is still relevant.
>This part is actually not used if the bins are given explicitly.

Ok, maybe I'm misunderstanding something but it sounds like the comment is still relevant but should be changes to something like `not used if bins are given explicitly`?